### PR TITLE
feat(uploader): including ImageUpload、VideoUpload and FileUpload

### PR DESCRIPTION
**Pull request by Alpsli**
[feat(image-upload): add image-upload component](https://github.com/ant-design/ant-design-mini/commit/71f792f55f3f9af979c229ac04157a70dd39a7f9)
[fix(minidev): update minidev version to resolve preview problem](https://github.com/ant-design/ant-design-mini/commit/28d96b7983da4d01e3efe5f198f44b8c12cb73b8)
[fix(image-upload): add preview default value for image-upload](https://github.com/ant-design/ant-design-mini/commit/3642932f81a95bb424d5cc63197f82bbcdc988c6)
[feat(image-upload): add css variable and change class name](https://github.com/ant-design/ant-design-mini/commit/919cdb7e17ba30b96eeae57c003bf93b1c39e28e)

### CODE REVIEW

File: src/FileUpload/index.axml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        <view class="{{item.status==='pending'?'amd-file-upload-brief-pending':''}}" a:if="{{item.status==='pending'}}">
+          <progress percent="{{item.progress}}" active-Color="#1677FF" />
+        </view>
+        <view a:elif="{{item.status === 'error'}}" class="amd-file-upload-error-status">上传错误</view>
+        <view a:else>
+          {{item.size}}
+        </view>
+      </view>
+      <view slot="extra" class="amd-file-upload-close-icon" a:if="{{item.status!=='pending' || item.progress === 100}}" data-delete-file-index="{{index}}" 
+        catchTap="onDeleteFile">
```
Comment (Simplyme0823) [inline src/FileUpload/index.axml:22]: 为什么是 catchTap

File: src/ImageUpload/index.axml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    <view class="amd-image-upload-demo-image-tip" />
+    <image class="amd-image-upload-demo-image" mode="scaleToFill" src="{{demoImage}}" style="{{`height:${height}px;width:${width}px`}}" />
+  </view>
+  <view a:for="{{fileList}}" class="amd-image-upload-show">
+    <view class="amd-image-upload-close" a:if="{{item.status !== 'pending'}}" onTap="onDeleteImage" data-delete-image-index="{{index}}" />
+    <view class="amd-image-upload-loading" a:else style="{{`height:${height}px;width:${width}px`}}">
+      <loading className="amd-image-upload-loading-icon" theme="light" text="上传中..." />
+    </view>
+    <slot name='label'>
+      <view class="amd-image-upload-image-label" a:if="{{item.description}}" style="{{`width:${width}px;top:calc(${height}px - 20px)`}}">
```
Comment (Simplyme0823) [inline src/ImageUpload/index.axml:12]: 这里做个单位判断吧，用户可能会写rpx ?

**Review by Simplyme0823 — COMMENTED** [review]
有两个很小的点

File: src/FileUpload/index.axml (hunk 1 above)
Comment (Alpsli) [reply src/FileUpload/index.axml:22]: 因为这个slot在list-item中，list-item绑定了点击事件，如果这里不阻止冒泡，就会触发

**Pull request closed without merging**